doc/python/imshow.md

```diff
@@ -399,6 +399,74 @@ for compression_level in range(0, 9):
 fig.show()
 ```
 
+### Exploring 3-D images and timeseries with `facet_col`
+
+*Introduced in plotly 4.13*
+
+For three-dimensional image datasets, obtained for example by MRI or CT in medical imaging, one can explore the dataset by representing its different planes as facets. The `facet_col` argument specifies along which axis the image is sliced through to make the facets. With `facet_col_wrap`, one can set the maximum number of columns. For image datasets passed as xarrays, it is also possible to specify the axis by its name (label), thus passing a string to `facet_col`.
+
+It is recommended to use `binary_string=True` for facetted plots of images in order to keep a small figure size and a short rendering time.
+
+See the [tutorial on facet plots](/python/facet-plots/) for more information on creating and styling facet plots.
+
+```python
+import plotly.express as px
+from skimage import io
+from skimage.data import image_fetcher
+path = image_fetcher.fetch('data/cells.tif')
+data = io.imread(path)
```

Comment:
```suggestion
from skimage import util
from skimage.data import cells3d
data = util.img_as_float(cells3d()[:, 1, :, :])  # grab only the nuclei
```

Reply:
Hum these changes have not been released in scikit-image, have they?

Reply:
You are right! The dev version tends to be *my* reference version, hence the confusion. Sorry, please disregard these suggestions. I'll delete them if it's straightforward to do so...